lib/puppet_x/puppetlabs/preview/api/documentation/preview-help.md

```diff
@@ -112,7 +112,8 @@ Note that all settings (such as 'log_level') affect both compilations.
   Specifies a file to read node-names from. If the file name is '-' file names are read
   from standard in. Each white-space separated sequence of characters is taken as a node name.
   This may be combined with additional nodes given on the command line. Duplicate entries (in given
-  file, or on command line) are skipped.
+  file, or on command line) are skipped. Deactivated and nodes for which there are no facts available
+  are skipped.
```

Comment:
1. Change "may" to "can."
2. "Duplicated entries, deactivated nodes, and nodes with no facts available are skipped."